developer_manual/core/theming.rst

```diff
@@ -1,166 +1,239 @@
+``'theme' => 'mytheme'`` to ``/config/config.php``.
 
 Structure
 =========
-The folder structure of a theme is exactly the same as the main ownCloud
-structure. You can override js files, images, translations and templates with
-own versions. CSS files are loaded additionally to the default files so you can
-override CSS properties. CSS files and the standard pictures that are used reside
-for example in /owncloud/core/ and /owncloud/settings/ in these sub folders:
-
-* css = style sheets
-* js = JavaScripts
-* img = images
-* l10n = translation files
-* templates = php and html template files
 
-.. _notes-for-updates:
+The folder structure of a custom theme is exactly the same as the main ownCloud structure.
+With it, you can override JavaScript files, images, user interface translations, and templates with your own custom versions.
 
+We recommend the following folder naming convention:
+
+============= ===========================
+Name          Purpose
+============= ===========================
+``css``       Style sheets
+``js``        JavaScripts
+``img``       Images
+``l10n``      Translation files
+``templates`` PHP and HTML template files
+============= ===========================
+
+.. note: 
+   Theme CSS files are always loaded *after* the default CSS files. So, your
+   theme will always override any default CSS properties. 
+
+.. _notes-for-updates:
 
 Notes for Updates
 =================
-It is not recommended to the user to perform adaptations inside the 
-folder ``/themes/example`` because files inside this folder might get
-replaced during the next ownCloud update process.
 
-During an update, files might get changed within the core and settings 
-folders. This could result in problems because your template files will 
-not 'know' about these changes and therefore must be manually merged with
-the updated core file or simply be deleted (or renamed for a test).
+It is **not** recommended to perform changes inside the folder ``/themes/example``. 
+Files inside this folder might get replaced during the next ownCloud update process.
+During an update, files might get changed within the core and settings folders. 
+This could result in problems, as your template files will not 'know' about these changes.
 
+As a result, they will need to be manually merged with the updated core file, or simply be deleted (or renamed in the case of tests).
 For example if ``/settings/templates/apps.php`` gets updated by a new
 ownCloud version, and you have a ``/themes/MyTheme/settings/templates/apps.php``
 in your template, you must merge the changes that where made within the update
 with the ones you did in your template.
 
-But this is unlikely and will be mentioned in the ownCloud release notes if it occurs.
+.. note: 
+   This is unlikely and will be mentioned in the ownCloud release notes if it
+   occurs.
+
+How to Change Images and the Default Logo
+=========================================
+
```

Comment:
I find this whole section reads a bit confusing. Since we are creating a theme here, we are technically not replacing any images, but we are overwriting the default ones with images we provide inside the theme.

For example if i want to replace `owncloud/core/img/star.svg` i would simply provide my custom one in `owncloud/apps/my-theme/core/img/star.svg`. Mimicing the original file's location inside my theme. This way i can overwrite templates, css, images and js files.

I would go with a really simple explanation here. In case some users don't understand a short description properly, we will receive feedback and can still give a more detailed explanation. But in general too much text tends to confuse more than it clarifies. (which i feel like it does here)